packages/data-models/appConfig.ts

```diff
@@ -195,7 +195,7 @@ const APP_UPDATES = {
 const ASSET_PACKS = {
   enabled: false,
   bucketName: "deployment_name",
```

Comment:
nit(non-blocking)
Probably better to leave this as an empty string so it can pass type-checks but also trigger a future warning if user has not specified

```suggestion
  bucketName: "",
```